azure/template.json

```diff
@@ -198,30 +275,49 @@
                                 "type": "Custom"
                             }
                         ]
+                    },
+                    "customHostName": {
+                        "value": "[parameters('apiCustomHostname')]"
+                    },
+                    "certificateThumbprint": {
+                        "value": "[reference('api-app-service-certificate').outputs.certificateThumbprint.value]"
```

Comment:
```suggestion
                        "value": "[if(greater(length(parameters('apiCustomHostname')), 0), reference('api-app-service-certificate', '2018-11-01').outputs.certificateThumbprint.value, '')]"
```

Reply:
As discussed, default value of empty string for custom host name parameters removed, the custom host names are known and should always be passed in